app/views/govuk_component/feedback.raw.html.erb

```diff
@@ -0,0 +1,110 @@
+<%
+  contact_govuk_path = "/contact/govuk"
+%>
+
+<div class="pub-c-feedback" data-module="feedback">
+  <div class="pub-c-feedback__prompt pub-c-feedback__js-show js-prompt" tabindex="-1">
+    <div class="js-prompt-questions">
+      <h3 class="pub-c-feedback__prompt-question">Is this page useful?</h3>
+
+      <%= link_to contact_govuk_path, {
+        class: 'pub-c-feedback__prompt-link pub-c-feedback__prompt-link--useful js-page-is-useful',
+        data: {
+          'track-category' => 'manualFeedbackForm',
+          'track-action' => 'ffYesClick'
+          },
+        } do %>
+          Yes <span class="visually-hidden">this page is useful</span>
+      <% end %>
+
+      <%= link_to contact_govuk_path, {
+        class: 'pub-c-feedback__prompt-link js-toggle-form js-page-is-not-useful',
+        data: {
+          'track-category' => 'manualFeedbackForm',
+          'track-action' => 'ffNoClick'
+          },
+          'aria-controls': 'page-is-not-useful',
+          'aria-expanded': false
+        } do %>
+          No <span class="visually-hidden">this page is not useful</span>
+      <% end %>
+
+      <%= link_to contact_govuk_path, {
+        class: 'pub-c-feedback__prompt-link pub-c-feedback__prompt-link--wrong js-toggle-form js-something-is-wrong',
+        data: {
+          'track-category' => 'manualFeedbackForm',
+          'track-action' => 'ffWrongClick'
+          },
+          'aria-controls': 'something-is-wrong',
+          'aria-expanded': false
+        } do %>
+          Is there anything wrong with this page?
+      <% end %>
+    </div>
+
+    <div class="pub-c-feedback__prompt-success js-prompt-success js-hidden" tabindex="-1">
+      Thanks for your feedback
```

Comment:
"Thank you for your feedback." seems more polite.

Reply:
Will change.